src/ResourceManagement/Compute/Microsoft.Azure.Management.V2.Compute/ComputeRoles.cs

```diff
@@ -7,7 +7,7 @@
 using System.Linq;
 using System.Threading.Tasks;
 
-namespace Microsoft.Azure.Management.V2.Compute
+namespace Microsoft.Azure.Management.Compute.Models
```

Comment:
you should have Fluent in the namespace. We do not want this libraries to collide with AutoRest generated SDK's